cluster/config-defaults.yaml

```diff
@@ -567,9 +567,8 @@ etcd_instance_count: "3"
 etcd_instance_type: "t3.medium"
 {{end}}
 
-etcd_docker_image: "registry.opensource.zalan.do/teapot/etcd-cluster:3.4.16-master-10"
```

Comment:
Is that still needed?

Reply:
It's not. We removed Docker from `etcd-on-ubuntu`.